cmd/bridge/main.go

```diff
@@ -197,14 +193,12 @@ func main() {
 		os.Exit(1)
 	}
 
-	capnslog.SetGlobalLogLevel(capnslog.INFO)
 	if *fLogLevel != "" {
-		llc, err := rl.ParseLogLevelConfig(*fLogLevel)
+		msg, err := glogSetter(convertLogLevel(*fLogLevel))
```

Comment:
Given this comment https://github.com/openshift/console-operator/pull/250/files#r352708637

I bet that the `bridge` binary will need a little func in the interim to handle either:

```bash
- command:
        - /opt/bridge/bin/bridge
        - --log-level=*=DEBUG
        # other flags...
```
or
```bash
- command:
        - /opt/bridge/bin/bridge
        - -v=4
        # other flags....
```

Tests will start failing once there is a mismatch.

Reply:
I just left the flag in console as --log-level and made a helper function that could handle 0-5 (Info levels) or the capnlog values.

Reply:
`klog` is expecting `-v=2`,`-v=4`, etc.
You'll want to swap to that.  However, in the interim, if `--log-level` is passed, you just want to allow it but not let the `bridge` binary crash.  `klog` can just used the default level then, and you can `klog.Info("DEPRECATED --log-level flag was passed. ignoring and using default log level.")

As long as this PR merges in the console before [#250](https://github.com/openshift/console-operator/pull/250) merges in the operator, you shouldn't break the merge queue.